### PR TITLE
Add ability to turn off badges for plugins

### LINKED ISSUE #247: Support turning off badges for plugins
e.g. to use `GitLabCI` but not have a badge in the readme

### PR DESCRIPTION
**Pull request by nickrobinson251**
- close #247 
- Takes the same approach as customising badge order (https://github.com/invenia/PkgTemplates.jl/pull/264)

### CODE REVIEW

File: test/reference.jl [hunk 1]
```diff
@@ -127,7 +127,7 @@ end
             GitLabCI(; coverage=false, extra_versions=[v"0.6"]),
             License(; name="ISC"),
             ProjectFile(; version=v"1"),
-            Readme(; inline_badges=true),
+            Readme(; inline_badges=true, badge_off=[Codecov]),
```
Comment (fchorney) [inline test/reference.jl:130]: If `badge_off` is set here for `Codecov` then shouldn't the `Codecov` not show up in the `WackyOptions/README.md` that was changed above?

Comment (nickrobinson251) [reply test/reference.jl:130]: oooo yes, sneaky! (i saw the output changed and didn't look closely at the ugly diff - oops)
Looks like badges get added in two goes... i will fix it!

Comment (nickrobinson251) [reply test/reference.jl:130]: hopefully fixed by https://github.com/invenia/PkgTemplates.jl/pull/268/commits/ac98aed85f69d5940aa77958848a4910d90f61cd :)

great catch

**Review by fchorney — APPROVED** [review]

**Comment by nickrobinson251** [comment]
@christopher-dG please can you merge and tag, or give me or @fchorney permissions to?

Thanks

**Comment by nickrobinson251** [comment]
bump :)

**Pull request merged**